libraries/AP_HAL_ChibiOS/hwdef/BlitzF7/README.md

```diff
@@ -0,0 +1,89 @@
+# iFlight BLITZ Whoop F7 AIO Flight Controller
+
+https://shop.iflight-rc.com/BLITZ-Whoop-F7-AIO-Pro1927
+
+The Blitz F7 AIO is a flight controller produced by [iFlight](https://shop.iflight-rc.com/).
+
+## Features
+
+ - MCU: BGA-STM32F745, 216MHz
+ - Gyro: DPS310
+ - 16Mb Onboard Flash
+ - BEC output: 5V 2.5A
+ - Barometer: DPS310 or None
+ - OSD: AT7456E
+ - 5 UARTS: (UART1, UART2, UART3, UART4, UART7)
+ - I2C for external compass.
+ - 5 PWM outputs (4 motors and 1 LED)
+
+## Pinout
+
+![BLITZ Whoop F7 AIO Board](blitz_f7_pinout.jpg "BLITZ Whoop F7 AIO")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+|Name|Pin|Function|
+|:-|:-|:-|
+|SERIAL0|COMPUTER|USB|
+|SERIAL1|RX1/TX1|UART1 (DJI connector, DMA-enabled)|
+|SERIAL2|TX2/RX2|UART2 (DJI connector RX, DMA-enabled, TX is on the back side of board)|
+|SERIAL3|TX3/RX3|UART3 (Alt RX, DMA-enabled)|
+|SERIAL4|TX4/RX4|UART4|
+|SERIAL7|TX7/RX7|UART7 (GPS)|
+
+## RC Input
+
+RC input is configured on the (UART2_RX/UART2_TX) pins which forms part of the DJI connector. It supports all RC protocols.
+
+## OSD Support
+
+The BLITZ Whoop F7 AIO supports OSD using OSD_TYPE 1 (MAX7456 driver).
+
+## PWM Output
+
+The BLITZ Whoop F7 AIO supports up to 4 PWM outputs. The pads for motor output ESC1 to ESC4 on the above diagram are for the 4 outputs. All 4 outputs support DShot as well as all PWM types.
+
+The PWM are in in two groups:
+
+ - PWM 1-4 in group1
+ - PWM 5   in group2
+
+Channels within the same group need to use the same output rate. If
+any channel in a group uses DShot then all channels in the group need
+to use DShot.
+
+## Battery Monitoring
+
+The board has a builtin voltage sensor. The voltage sensor can handle up to 6S
+LiPo batteries.
+
+The correct battery setting parameters are:
+
+ - BATT_MONITOR 4
+ - BATT_VOLT_PIN 12
+ - BATT_VOLT_MULT around 10.9
+ - BATT_CURR_PIN 13
+ - BATT_CURR_MULT around 50
+
+These are set by default in the firmware and shouldn't need to be adjusted
+
+## Compass
+
+The BLITZ Whoop F7 AIO does not have a builtin compass, but you can attach an external compass to I2C pins.
+
+## NeoPixel LED
+
+The board includes a NeoPixel LED on the underside which is pre-configured to output ArduPilot sequences. This is the fifth PWM output.
+Currently this output is not functional.
```

Comment:
looks to me to be defined as functional and setup by default for NEOPIXEL

Reply:
Removed